cypress.config.ts

```diff
@@ -0,0 +1,16 @@
+import { defineConfig } from "cypress";
+
+export default defineConfig({
+  component: {
+    devServer: {
+      framework: "react",
+      bundler: "vite",
+    },
+  },
+
```

Comment:
Hello, regarding the issue of failing to resolve local backend at localhost:8080 when running the cypress tests through the cypress managed browser:

There seems to be not so well documented option in cypress configuration file that allows to explicitly define hostmapping to ip addresses by adding the following parameter ` hosts: { localhost: "127.0.0.1" },` .e.g in the existing config file:
```typescript
export default defineConfig({
  component: {
    devServer: {
      framework: "react",
      bundler: "vite",
    },
  },
  hosts: { localhost: "127.0.0.1" },
 });
```

This parameter is not described in the official docs but is evident in the typescript declarations of the configuration see ref: https://github.com/cypress-io/cypress/blob/b71ce4422f87ac5ed74d60debd20d5e450d0128b/cli/types/cypress.d.ts#L3240-L3243

Reply:
Also i think it is safe for the time being to remove the `e2e` section:
```
e2e: {
    setupNodeEvents(on, config) {
      // implement node event listeners here
    },
  },
```
from the config file until we have the need of specifically implementing the node event listeners there

Reply:
Thank you for looking into this issue, this solution looks good to me! I will make the changes and check with Daniel on Monday if this will also solve the issue on his setup. 

Agreed that removing the empty node event listener setup for now is better.

Reply:
Thank you Fjodorv for the prompt response 👍 Take your time to test it on your local environments and we can revisit this PR next week

Reply:
If its possible also, please rebase this branch according to the latest devel

Reply:
> If its possible also, please rebase this branch according to the latest devel
Done!

Reply:
> Thank you Fjodorv for the prompt response 👍 Take your time to test it on your local environments and we can revisit this PR next week

Sounds good, I'll let you know next week if it worked.

Reply:
Hello Fjodorv, does the change work on your environments?

Reply:
Hi Kostas, sorry for the late response. We've just finished testing it on the other environments and it works! We did need to make a small change in that we needed to add 127.0.0.1 as the host in the vite.config.ts file.

Reply:
I believe this can also be set with the --host flag so it shouldn't require a change:
https://vitejs.dev/config/server-options.html